index3.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8"/>
+		<title>myheritage/ethnicity-estimate</title>
+		<link href="Ethnic_origin.css" rel="stylesheet"/>
+		<link href="reset.css" rel="stylesheet"/>
+	</head>
+	<body>
+		<header>
+			<div class="logo_image">
+				<img src="assets/logo.jpg">
+			</div>
+				<div class="menu">
+					<span>Дерево</span>
+					<span>Фотографії</span>
+					<span>Пошук</span>
+					<span>ДНК</span>
+					<span>Допомога</span>
+				</div>
+		</header>
+		<main>
+			<section class="Ethnic_origin">
+				<div class="Ethnic_origin_title">Етнічне походження
+				</div>
+				<p>
+					<div class="Ethnic_origin_text">
+					Отримайте<br>розгалужений розподіл<br>вашого походження — <br><b>2 114 регіонів</b>
+					</div>
+				</p>
+					<div class="Ethnic_origin_button">
+						<button class="Ethnic_origin_button_design">
+							<a href="">Замовити зараз</a>
+						</button>
+					</div>
+			</section>
+			<section class="percentage_distribution">
+				<div class="percentage_distribution_title">Дізнайтесь про своє<br>
+															етнічне походження
+				</div>
+				<div class="percentage_distribution_text">Результати тесту вашої ДНК від MyHeritage включають<br>
+														визначення етнічного походження: процентний розподіл<br>
+														вашого походження, що показує походження ваших<br>
+														предків серед 42 етнічних груп, включаючи японську,<br>
+														ірландську, італійську, єврейську та багатьох інших. Ви<br>
+														знайдете список регіонів, з яких походять ваші предки<br>
+														— з обох сторін вашої родини, — а також відсотковий <br>
+														склад вашої ДНК, що бере початок із кожного регіону.
+					<img class="line1" src="assets/line.png">
+				</div>
+			</section>
+			<section class="Genetic_groups">
+				<div class="Genetic_groups_title">Отримайте детальне<br>
```

Comment:
Please avoid using `div`s when it is effectively 'headings' like `h1-h5`. And replace such `div`s at all similar places across the page

Reply:
`div` in most case just a 'container' to wrap 2 to many elemnets